lib/plug/adapters/cowboy2.ex

```diff
@@ -126,15 +126,30 @@ defmodule Plug.Adapters.Cowboy2 do
   ## Examples
 
   Assuming your Plug module is named `MyApp` you can add it to your
-  supervision tree by using this function:
+  supervision tree by using this function
+
 
       children = [
-        {Plug.Adapters.Cowboy2, scheme: :http, plug: MyApp, options: [port: 4040]}
```

Comment:
The previous code is actually correct. It contains the new child_spec API that comes as part of Elixir v1.5. You don't have to call child_spec directly.

Reply:
Yeah, I know.  It is just weird to have the child_spec() function and then not need the function in docs for that function.

I'm ok either way, of course.  It is you guys project.